<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: #B0C4DE;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:#D3D3D3 ;
border:1px solid #000000;
border-radius: 10px;
transition: ease ;
duration: 500;

}

QPushButton:hover{
background-color:#87CEEB;
border:1px solid #000000;
border-radius: 10px;


}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Odpowiedz A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Odpowiedz B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Odpowiedz C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Odpowiedz D</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ProcA">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser{
background-color:#D3D3D3 ;
border:1px solid #000000;
border-radius: 10px;
transition: ease ;
duration: 500;

}
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ProcB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser{
background-color:#D3D3D3 ;
border:1px solid #000000;
border-radius: 10px;
transition: ease ;
duration: 500;

}
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ProcC">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser{
background-color:#D3D3D3 ;
border:1px solid #000000;
border-radius: 10px;
transition: ease ;
duration: 500;

}
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ProcD">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser{
background-color:#D3D3D3 ;
border:1px solid #000000;
border-radius: 10px;
transition: ease ;
duration: 500;

}
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
